lib/workers/repository/dependency-dashboard.ts

```diff
@@ -322,14 +322,14 @@ export async function ensureDependencyDashboard(
       'This repository currently has no open or pending branches.\n\n';
   }
 
-  issueBody += PackageFiles.getDashboardMarkdown(config);
+  // fit the detected dependencies section
+  const footer = getFooter(config);
+  issueBody += PackageFiles.getTruncatedMarkdown(
+    config,
+    60000 - issueBody.length - footer.length
```

Comment:
Where does `60000` come from?

Reply:
That sounds like a specific platform issue body limit, such as GitHub

Reply:
Yeah, it's a magic number. I would extract it to a constant and document it.

Reply:
added an exported const in `lib/modules/platform/github/index.ts`